tests/e2e/logging/test_openshift-logging.py

```diff
@@ -0,0 +1,148 @@
+
+    * The teardown will uninstall cluster-logging from the cluster
+
+    """
+
+    def finalizer():
+        teardown(sc_obj, cbp_obj)
+
+    request.addfinalizer(finalizer)
+
+    # Deploys elastic-search operator on the project openshift-operators-redhat
+    obj.create_namespace(yaml_file=constants.EO_NAMESPACE_YAML)
+    assert obj.create_elasticsearch_operator_group(
+        yaml_file=constants.EO_OG_YAML,
+        resource_name='openshift-operators-redhat'
+    )
+    assert obj.set_rbac(
+        yaml_file=constants.EO_RBAC_YAML, resource_name='prometheus-k8s'
+    )
+    assert obj.create_elasticsearch_subscription(constants.EO_SUB_YAML)
+
+    # Deploys cluster-logging operator on the project openshift-logging
+    obj.create_namespace(yaml_file=constants.CL_NAMESPACE_YAML)
+    assert obj.create_clusterlogging_operator_group(
+        yaml_file=constants.CL_OG_YAML
+    )
+    assert obj.create_clusterlogging_subscription(
+        yaml_file=constants.CL_SUB_YAML
+    )
+
+    # Creates instance for cluster-logging
+    cbp_obj = helpers.create_ceph_block_pool()
+    sc_obj = helpers.create_storage_class(
+        interface_type=constants.CEPHBLOCKPOOL,
+        interface_name=cbp_obj.name,
+        secret_name=constants.DEFAULT_SECRET,
+        reclaim_policy="Delete"
+    )
+    assert sc_obj, f"Failed to create storage class"
+    assert obj.create_instance_in_clusterlogging(sc_name=sc_obj.name)
+
+    # Check the health of the cluster-logging
+    assert obj.check_health_of_clusterlogging()
+
+
+def teardown(sc_obj, cbp_obj):
+    """
+    The teardown will uninstall the openshift-logging from the cluster
+    """
+    sc_obj.delete()
+    cbp_obj.delete()
+    uninstall_cluster_logging()
+
+
+@pytest.mark.usefixtures(
+    test_fixture.__name__
+)
+class TestLogging_in_EFK_stack(E2ETest):
+    """
+    The class contains the testcases related to openshift-logging
+    """
+    @pytest.fixture()
+    def create_pvc_and_deploymentconfig_pod(self, request, pvc_factory):
+        """
+
+        """
+        def finalizer():
+            helpers.delete_deploymentconfig(pod_obj)
+
+        request.addfinalizer(finalizer)
+
+        # Create pvc
+        pvc_obj = pvc_factory()
+
+        # Create service_account to get privilege for deployment pods
+        sa_name = helpers.create_serviceaccount(pvc_obj.project.namespace)
+
+        helpers.add_scc_policy(sa_name=sa_name.name, namespace=pvc_obj.project.namespace)
+
+        pod_obj = helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_obj.name,
+            namespace=pvc_obj.project.namespace,
+            sa_name=sa_name.name,
+            dc_deployment=True
+        )
+        helpers.wait_for_resource_state(resource=pod_obj, state=constants.STATUS_RUNNING)
+        return pod_obj, pvc_obj
+
+    @pytest.mark.polarion_id("OCS-657")
+    @tier1
+    @retry(ModuleNotFoundError, 6, 300, 3)
```

Comment:
does it wait for 300 seconds? why such a long time?

Reply:
For the new-project to appear in the EFK stack it took some time. Sometimes within 2, 3 minutes, sometimes a bit later than that.

Reply:
I see. okay.